libs/entry-components/styles/modules/components/buttons/_radio.scss

```diff
@@ -7,8 +7,12 @@ $radio-circles-selector: '.mdc-radio__outer-circle, .mdc-radio__inner-circle';
 	.mat-mdc-radio-group .mat-mdc-radio-button {
 		&.mat-mdc-radio-checked .mdc-radio {
 			&:not(:disabled):active #{$radio-selector}, #{$radio-selector} {
-				#{$radio-circles-selector} {
-					border-color: map.get($theme, 'general', 'buttons', 'radio', 'background');
+				$radio-border: map.get($theme, 'general', 'buttons', 'radio', 'background');
+
```

Comment:
For all these files with if check, if should go to the most upper layer possible.
Right now we're still generating CSS and selectors but without any properties. We shouldn't generate anything if value is null for all the checks.